core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -1870,7 +1870,6 @@ private RecordCursorFactory generateTableQuery(
                         // we can only deal with 'order by symbol, timestamp' at best
                         // skip this optimisation if order by is more extensive
                         final int columnIndex = metadata.getColumnIndexQuiet(model.getOrderByAdvice().getQuick(0).token);
-                        assert columnIndex > -1;
```

Comment:
we should probably keep that - advice must not have unresolvable columns

Reply:
done